### PR TITLE
feature: python - starlite

### PR DESCRIPTION
**Pull request by raphaelauv**

### CODE REVIEW

File: python/litestar/requirements.txt
Comment (waghanza) [inline python/litestar/requirements.txt:1]: ℹ️  version updates will be made by `renovatebot`

File: python/litestar/config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+framework:
+  website: litestar.dev
+  version: "2.0.0rc1"
```
[the remark this answers is not in the document]
Comment (raphaelauv) [reply python/litestar/config.yaml:3]: 2.0.0 does not exist yet

File: python/litestar/requirements.txt
Comment (raphaelauv) [reply python/litestar/requirements.txt:1]: what do you mean ? what should I change ?

Comment (waghanza) [reply python/litestar/requirements.txt:1]: nothing, just to inform you that a PR will be automagically created when a new release of starlite is out 🎉

Comment (raphaelauv) [reply python/litestar/requirements.txt:1]: okay , thanks

**Comment by waghanza** [comment]
We are going to introduce `litestar` in here @Goldziher @provinzkraut @peterschutt. Are you OK with such a thing ?

**Comment by provinzkraut** [comment]
Sure thing

**Comment by Goldziher** [comment]
> We are going to introduce `litestar` in here @Goldziher @provinzkraut @peterschutt. Are you OK with such a thing ?

Of course, go ahead! We are about a week away from the official release of V2, so it would be neat.

File: python/litestar/config.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (waghanza) [reply python/litestar/config.yaml:3]:
```suggestion
  version: "2.0"
```

I undertand that 2.0.0 does not exist yet (but will be). I think it is better to show X.Y on results + anticipate update in here (I mean next step after rc is 2.0.0)

File: python/litestar/config.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+  engines:
+    - uvicorn
```
Comment (waghanza) [inline python/litestar/config.yaml:6]: ⚠️ Actually results shows only the first engine, here `unicorn`. Is this the recommended way to deploy in **production** ?

Comment (raphaelauv) [reply python/litestar/config.yaml:6]: starlite is ASGI , unicorn is one of the "best" ASGI web server 

https://docs.litestar.dev/2/usage/applications.html#startup-and-shutdown

Comment (raphaelauv) [reply python/litestar/config.yaml:6]: and intern benchmark use unicorn -> https://docs.litestar.dev/2/benchmarks.html#methodology

**Comment by waghanza** [comment]
> Are you only measuring text responses? No parameter parsing, json, etc?

Not for now, but it is in the roadmap

Next step : 
- Run on cloud (avoid using my personal workstation)
- Make a more robust scenario (something that represent a typical backend workload)

Thos 2 ideas could ba parallelized, I will gather ideas @Goldziher

**Review by waghanza — APPROVED** [review]

**Comment by Goldziher** [comment]
> > Are you only measuring text responses? No parameter parsing, json, etc?
> 
> Not for now, but it is in the roadmap
> 
> Next step : 
> - Run on cloud (avoid using my personal workstation)
> - Make a more robust scenario (something that represent a typical backend workload)
> 
> Those 2 ideas could ba parallelized, I will gather ideas @Goldziher 
> 
> ----
> 
> EDIT : For now results are update each monday

Aight. You can also look at our API benchmarks. @provinzkraut did a really amazing job there. It's python only, but very thorough.

**Pull request merged**